### PR TITLE
Add StackName to Outputs. Useful when used as nested stack.

### PR DESCRIPTION
**Pull request by guizmaii**

### CODE REVIEW

**Comment by michaelwittig** [comment]
Can you elaborate on the use case if you use nested stacks?

I assume that you have a template with `AWS::CloudFormation::Stack` resources. One of the resources is a vpc stack. To create e.g. the wordpress stack you need to pass in the vpc stack name, that's why you need this output? correct?

**Comment by guizmaii** [comment]
Hi,

Yes I can. 

Here is what I'm doing:
```
{
  ...
  
  "Resources": {
    "Network": {
      "Type": "AWS::CloudFormation::Stack",
      "Properties": {
        "TemplateURL": "https://s3-eu-west-1.amazonaws.com/widdix-aws-cf-templates-releases-eu-west-1/v4.7.0/vpc/vpc-3azs.yaml",
        "Parameters": {
          "ClassB": 0
        }
      }
    },

    "NatA": {
      "Type": "AWS::CloudFormation::Stack",
      "DependsOn": "Network",
      "Properties": {
        "TemplateURL": "https://s3-eu-west-1.amazonaws.com/widdix-aws-cf-templates-releases-eu-west-1/v4.7.0/vpc/vpc-nat-gateway.yaml",
        "Parameters": {
          "ParentVPCStack": // Here I need the "Network" StackName
          "SubnetZone": "A"
        }
      }
    },

   ...
}
```

The `"Network"` `StackName` is a generated value.
If I understand well what is `ParentVPCStack`, I need to pass this `"Network"` `StackName` as value but I don't find how to get this generated `StackName`

File: vpc/vpc-2azs.yaml [hunk 1]
```diff
@@ -220,6 +220,9 @@ Outputs:
   TemplateID:
     Description: 'cloudonaut.io template id'
     Value: 'vpc/vpc-2azs'
+  StackName:
+    Description: 'Nested stack name'
```
Comment (michaelwittig) [inline vpc/vpc-2azs.yaml]: could you change 'Nested stack name' to 'Stack name' only? It's not really nested until you put it into another stack.
Same applies to the other templates

Comment (guizmaii) [inline vpc/vpc-2azs.yaml]: Yes, I'll do that

**Comment by guizmaii** [comment]
`Do you have time to add this output to all the other templates as well? If not, I will do.` -> yes

**Comment by guizmaii** [comment]
Do you want an `export` too ?

File: vpc/vpc-2azs.yaml (hunk 1 above)
Comment (guizmaii) [inline vpc/vpc-2azs.yaml]: Done

**Comment by guizmaii** [comment]
Is it possible for you tu publish a `v4.8.0` or `v4.7.1` on S3 ? It'll unblock us in our work.

**Comment by michaelwittig** [comment]
yes, the test suite will now run. This usually takes 3 hours... after that I can create a new release.

**Comment by guizmaii** [comment]
Ok thanks !

**Comment by guizmaii** [comment]
Are the tests ok ?

**Comment by michaelwittig** [comment]
released: https://github.com/widdix/aws-cf-templates/releases/tag/v4.8.0

**Comment by guizmaii** [comment]
Thanks 🙂

**Pull request merged**